pkg/maps/eppolicymap/eppolicymap.go

```diff
@@ -100,9 +100,6 @@ func (v *EPPolicyValue) GetValuePtr() unsafe.Pointer { return unsafe.Pointer(v)
 func (k EndpointKey) NewValue() bpf.MapValue { return &EPPolicyValue{} }
 
 func writeEndpoint(keys []*lxcmap.EndpointKey, fd int) error {
-	if option.Config.SockopsEnable == false {
-		return nil
-	}
```

Comment:
> @qmonnet, should I also delete some functions from `pkg/maps/eppolicymap/eppolicymap_test.go` that contains _SockopsEnable_?

Likely, but wait there's more...

The change in `writeEndpoint` here looks in fact incorrect. When `option.Config.SockopsEnable` is `false`, the function should do nothing. Now that this option is being removed, i.e. always implicitly `false`, the function should always do nothing. So `writeEndpoint()` and `WriteEndpoint()` should just do nothing, and in fact, be removed.

`CreateWithName()` and `CreateEPPolicyMap()` in the same file are also no longer used, given that their usage were conditioned by the option being `true`.

So yeah, it's likely you can remove some functions in this test file, and all the rest of the test file, and the `eppolicymap` package with it :).